omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -129,7 +129,7 @@
         {server_version, "<%= node['private_chef']['version'] %>"},
         {health_ping_timeout, <%= node['private_chef']['opscode-erchef']['health_ping_timeout'] %>},
         {health_ping_modules, [
-            <% if node['private_chef']['data_collector']['root_url'] %>
+            <% if node['private_chef']['data_collector']['root_url'] && node['private_chef']['data_collector']['health_check'] == true %>
```

Comment:
I might just rely on the truthyness of the node['private_chef']['data_collector']['health_check'] variable, and not test explicitly against true.

Reply:
@markan, added RSpec tests (for parsing of config) and switched to be truthy. Let me know if you need somethign else for tests.